contrib/babelfishpg_tsql/sql/object_definition_tsql.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE sys.babelfish_view_def (
+	dbid SMALLINT NOT NULL,
+	schema_name sys.SYSNAME NOT NULL,
+	object_name sys.SYSNAME NOT NULL,
+	definition sys.NTEXT,
+	is_ansi_nulls_on sys.BIT,
```

Comment:
what if we need more flags in future? will be tricky to upgrade existing customer. Will it make more sense to have a 64 bit field so that we can cover up to 64 flags without an upgrade

Reply:
Addressed introduced flag_validity(to track whether we're use corresponding bit in flag_values to represent any flag like ansi_nulls, quoted_identifier etc. related to view) and flag_values(to store actual values) int64 values in catalog.